### PR TITLE
Minitest for school.rb

### PR DESCRIPTION
**Pull request by badgerbreezy**

### CODE REVIEW

**Comment by KathleenScriver** [comment]
Definitely want to see more commits throughout your work. Next challenge, in your pull request, please title it your name. Thank you! Helps us out a ton. :)

File: lib/school.rb [hunk 1]
```diff
@@ -0,0 +1,25 @@
+class School
+  def initialize(start_time, hours_in_school_day)
+    @start_time = start_time
+    @hours_in_school_day = hours_in_school_day
+    @student_names = []
+  end
+
+  def start_time
```
Comment (KathleenScriver) [inline lib/school.rb:8]: This does work, however, remember in Ruby, we get the hand tool of the attr_reader to help us with these methods.

File: lib/school.rb [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  def hours_in_school_day
+    @hours_in_school_day
+  end
+
+  def student_names
+    @student_names
+  end
+
+  def add_student_name(student_name)
+    @student_names << student_name
```
Comment (KathleenScriver) [inline lib/school.rb:21]: Great job adding these names to the array.

**Comment by badgerbreezy** [comment]
Thank you Kat, I did end up working through the whole thing and finishing it, can I commit my changes to github and have you review it to see how I did? I think a lot of my problems were grasping the ideas of minitest and pry, but after spending some time reviewing material and chatting with classmates, I think I have a much better understanding of what their purpose is and how to use them.

**Comment by badgerbreezy** [comment]
Okay nevermind looks like we are going through it step by step anyway, thanks!

**Pull request closed without merging**